examples/compose/plot_column_transformer_mixed_types.py

```diff
@@ -3,21 +3,22 @@
 Column Transformer with Mixed Types
 ===================================
 
-This example illustrates how to apply different preprocessing and
-feature extraction pipelines to different subsets of features,
-using :class:`sklearn.compose.ColumnTransformer`.
-This is particularly handy for the case of datasets that contain
-heterogeneous data types, since we may want to scale the
-numeric features and one-hot encode the categorical ones.
-
-In this example, the numeric data is standard-scaled after
-mean-imputation, while the categorical data is one-hot
-encoded after imputing missing values with a new category
-(``'missing'``).
-
-Finally, the preprocessing pipeline is integrated in a
-full prediction pipeline using :class:`sklearn.pipeline.Pipeline`,
-together with a simple classification model.
+This example illustrates how to apply different preprocessing and feature
+extraction pipelines to different subsets of features, using
+:class:`sklearn.compose.ColumnTransformer`. This is particularly handy for the
+case of datasets that contain heterogeneous data types, since we may want to
+scale the numeric features and one-hot encode the categorical ones.
+
+In this example, the numeric data is standard-scaled after mean-imputation,
+while the categorical data is one-hot encoded after imputing missing values
+with a new category (``'missing'``).
+
+In addition, we show two different ways to dispatch the columns to the
```

Comment:
Note: This is the only paragraph modified here. The other introduction paragraphs were only wrapped to fill the 79 characters' length.